packages/regex/match_iter.pony

```diff
@@ -0,0 +1,38 @@
+class MatchIterator is Iterator[Match]
```

Comment:
per the pony file naming conventions this file shoudl be called match_interator.pony